mailexporter.go

```diff
@@ -294,10 +265,10 @@ func deleteMail(m email) {
 	promlog.Debug("rm ", m.filename)
 }
 
-// lateMail logs mails that have been so late that they timed out
-func lateMail(m email) {
```

Comment:
Good idea renaming this function, but I'd prefer `handleLateMail`, as this function also performs filesystem-operations. Can you change that name?